webpack.config.js

```diff
@@ -107,13 +107,17 @@ const wordpressExternals = ( context, request, callback ) =>
  * @see {@link https://webpack.js.org/configuration/configuration-types/#exporting-a-function}
  *
  * @param {object}  env                              additional config options
+ * @param {string}  env.cssModules                   whether to use CSS modules
  * @param {boolean} env.externalizeWordPressPackages whether to bundle or extern the `@wordpress/` packages
- * @param {object}  argv                             given by webpack?
+ * @param {object}  argv                             Describes the options passed to webpack
  *
  * @return {object}                                  webpack config
  */
-// eslint-disable-next-line no-unused-vars
-function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false } = {}, argv ) {
+function getWebpackConfig(
+	{ cssFilename, cssModules = false, externalizeWordPressPackages = false } = {},
+	// eslint-disable-next-line no-unused-vars
+	argv
```

Comment:
Let's just remove this, it's unused and eslint always complains.

Reply:
Aye!

🔪 https://github.com/Automattic/wp-calypso/pull/26996 🔪